noir-projects/aztec-nr/aztec/src/macros/mod.nr

```diff
@@ -15,6 +15,10 @@ use dispatch::generate_public_dispatch;
 
 /// Marks a contract as an Aztec contract, generating the interfaces for its functions and notes, as well as injecting
 /// the `compute_note_hash_and_optionally_a_nullifier` function PXE requires in order to validate notes.
+///
+/// Note: This is a module annotation, so the returned quote gets injected inside the module (contract) itself.
+/// The difference between a module and a regular struct or functionn annotation is that module annotations inject
```

Comment:
The first line of the note is fine, the second one I feel like it's repeating noir docs

Reply:
Nuked that in https://github.com/AztecProtocol/aztec-packages/pull/8901/commits/352f8a13894eb69b0619b56788426a306fd581fd